x-pack/plugins/observability_solution/synthetics/server/synthetics_service/get_api_key.ts

```diff
@@ -19,20 +19,24 @@ import { checkHasPrivileges } from './authentication/check_has_privilege';
 
 export const syntheticsIndex = 'synthetics-*';
 
-export const serviceApiKeyPrivileges = {
-  cluster: ['monitor', 'read_ilm', 'read_pipeline'] as SecurityClusterPrivilege[],
-  indices: [
-    {
-      names: [syntheticsIndex],
-      privileges: [
-        'view_index_metadata',
-        'create_doc',
-        'auto_configure',
-        'read',
-      ] as SecurityIndexPrivilege[],
-    },
-  ],
-  run_as: [],
+export const getServiceApiKeyPrivileges = (isServerless?: boolean) => {
```

Comment:
I reason why to accept undefined (`isServerless?:`) here?

Reply:
https://github.com/elastic/kibana/pull/178897#discussion_r1530564989